app/controllers/application_controller.rb

```diff
@@ -190,4 +190,12 @@ def redirect_old_urls
   def signed_in?
     !current_user.nil?
   end
+
+  def check_email_verified
+    unless current_user.nil?
+      unless current_user.is_verified
+        flash.now[:notice] = "Your email is not verified. Click <a href='../verify-email'>here</a> to verify email"
+      end
```

Comment:
Have you created any migration or is there a plan to create one to insert is-verified  as true for existing users of PL? Thanks!

Reply:
Okay so I need your suggestion on that. One thing we can do is, we would force only those users to verify email, who have registered after a certain date, Or we can write a migration to verify already registered users, Or we can make the current users to verify their email as well, this would help to get rid of spammers that are currently registered with fake email ids.

Reply:
Actually, the spammers are banned as soon as they try to post something bad, so if we consider ideal case that there are no spammers out there for now, then we would be prompting only valid users. Also, is there any condition that unverified users can't post or something so as to prevent spam?

Reply:
No, we have a different issue for that. So as of now there is no restriction on any user !

Reply:
Okay in case we are sure that there are no spammers as of now, then we can write a migration to make is_verified of users true